### DISCUSSION TITLE
How To Undo A Commit?

### DISCUSSION THREAD

**Discussion by Engine44** [community/community#23648] [New to GitHub]
<p>I just revised a few of the files in my repository with a Commit.  I would like to return to the previous version of my repository.  How do I do that? Thanks.</p>

**Comment by Engine44** [comment, marked as answer]
<p>I figured out how to do it.  Thanks.</p>

**Comment by VALERYRAMIREZ** [comment]
<p>Hi <a class="mention" href="/engine44">@engine44</a>, how are you?, how did you do it?.</p>
<p>Thanks in advance for the help</p>
